spec/requests/packages_spec.rb

```diff
@@ -464,24 +463,32 @@
         end
       end
 
+      #   # NOTE: alphabetical sorting tests for titles are currently limited
+      #   # due to a limitation in RM API. Cannot compare sorted results with
+      #   # a sorted list (some titles appear out of order).
+      #   # Additionally the same search with different sorts yields different counts.
+      #   # Title sorting is not being invoked from UI until
+      #   # sort limitation and a title count difference is resolved.
+
       describe 'with passing a sort of name' do
         before do
           VCR.use_cassette(
             'get-packages-related-resources-query-with-name-sort'
           ) do
-            get '/eholdings/packages/19-6581/resources/'\
-                 '?q=bioworld%20week&sort=name',
+            get '/eholdings/packages/19-6581/resources?'\
+                 'filter[name]=technology&sort=name',
                 headers: okapi_headers
           end
         end
 
         let!(:json_query_with_name_sort) { Map JSON.parse response.body }
 
-        it 'returns a list sorted by name' do
+        xit 'returns a list sorted by name' do
```

Comment:
an additional `x` instead of `it`

Reply:
xit skips the test and doesn't run the expectations. It leaves the test as `pending` in the test run. So future devs will know something is pending. I added a comment as to why it was skipped.

Reply:
sounds good 👍